### PR TITLE
Invite and Login UX

### PR DESCRIPTION
**Pull request by sbauch**
1. conditional email template for first vs. subsequent users, and update email copy, from address, etc
2. push login and account verification down to client app - changes how we authenticate the client from cookie to auth header

Screen Shot 2020-12-26 at 9 03 17 AM
Screen Shot 2020-12-26 at 9 02 45 AM

### CODE REVIEW

File: views/rodauth/verify-account-email.erb [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-Visit <%= rodauth.verify_account_email_link %> to specify a password and verify the account.
\ No newline at end of file
+A colleague has invited you to start using Osso to manage SAML SSO.
+
+Visit <%= rodauth.verify_account_email_link %> to specify a password and create your account.
+
+You can learn more about Osso at ossoapp.com
+
+Thanks,
+The Osso Team
```
Comment (sbauch) [inline views/rodauth/verify-account-email.erb]: Invitations for any user other than first

File: views/rodauth/verify-first-account-email.erb [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
\ No newline at end of file
+Your Osso instance is deployed and ready to start using.
+
+Please visit <%= rodauth.verify_account_email_link %> to create your admin account and specify a password. You'll be able to invite teammates to your instance once you've signed in.
+
+Review our documentation for integrating Osso into your application.
+
+Cheers,
+Sam, Sam and Michael
+The Osso Team
```
Comment (sbauch) [inline views/rodauth/verify-first-account-email.erb]: invitation for first user once the app is done deploying

File: views/rodauth/verify-first-account-email.erb [hunk 3]
```diff
@@ -1,3 +1,11 @@
-Someone has created an account with this email address. If you did not create this account, please ignore this message.
+Hello!
```
Comment (sccarmichael) [inline views/rodauth/verify-first-account-email.erb:1]:
```suggestion
Hello,
```

File: views/rodauth/verify-first-account-email.erb [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
 
-If you created this account, please go to <%= rodauth.verify_account_email_link %> to verify the account and specify a password.
\ No newline at end of file
+Your Osso instance is deployed and ready to start using.
```
Comment (sccarmichael) [inline views/rodauth/verify-first-account-email.erb:3]:
```suggestion
Great news! Your Osso instance is deployed and ready for use.
```

File: views/rodauth/verify-first-account-email.erb [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+
+Please visit <%= rodauth.verify_account_email_link %> to create your admin account and specify a password. You'll be able to invite teammates to your instance once you've signed in.
```
Comment (sccarmichael) [inline views/rodauth/verify-first-account-email.erb:5]:
```suggestion
Please visit <%= rodauth.verify_account_email_link %> to create your admin account and specify a password. Once that's done you'll be able to invite teammates to your Osso instance.
```

File: views/rodauth/verify-first-account-email.erb [hunk 6]
```diff
@@ -1,3 +1,11 @@
...
+
+Review our documentation for integrating Osso into your application.
```
Comment (sccarmichael) [inline views/rodauth/verify-first-account-email.erb:7]: @sbauch is this meant to include a link to docs?

File: views/rodauth/verify-first-account-email.erb [hunk 7]
```diff
@@ -1,3 +1,11 @@
...
+
+Cheers,
```
Comment (sccarmichael) [inline views/rodauth/verify-first-account-email.erb:9]:
```suggestion
Thanks for choosing Osso,
```

File: views/rodauth/verify-first-account-email.erb (hunk 6 above)
Comment (sbauch) [reply views/rodauth/verify-first-account-email.erb:7]: prob should huh, ill get it sorted

**Comment by sccarmichael** [comment]
Looking very close! Punchlist attached: 
image

**Comment by sccarmichael** [comment]
This is the code for the ghost button: `<button type="button" class="ant-btn ant-btn-primary ant-btn-background-ghost" ant-click-animating-without-extra-node="false"><span>Primary</span></button>`

**Review by sccarmichael — APPROVED** [review]
It's a beautiful thing, thanks for the custom CSS on the button!

**Pull request merged**